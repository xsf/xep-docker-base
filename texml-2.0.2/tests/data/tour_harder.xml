<TeXML>
  <cmd name="documentclass">
    <opt>a4paper</opt>
    <parm>article</parm>
  </cmd>
  ....
  <env name="document">
    Hello, <group><cmd name="it"/>World</group>!
  </env>
</TeXML>
