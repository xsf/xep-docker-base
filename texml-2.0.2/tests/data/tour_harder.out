\documentclass[a4paper]{article} ....
\begin{document}
Hello, {\it{}World}!
\end{document}
